config/application.rb

```diff
@@ -11,7 +11,7 @@ class Application < Rails::Application
 
     # allow nested structure in Models directory without additional namespacing
     # from http://stackoverflow.com/questions/18934115/rails-4-organize-rails-models-in-sub-path-without-namespacing-models
-    #config.autoload_paths += Dir[Rails.root.join('app', 'models', '{**}')]
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
The idea was to their use without having to namespace. I think this isn't necessary right now, since we don't have a very nested structure and since the namespacing is actually indicative of what the models are for.